docs/diagrams/Untitled Diagram.xml

```diff
@@ -0,0 +1 @@
+<mxfile userAgent="Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/61.0.3163.100 Safari/537.36" version="7.5.6" editor="www.draw.io" type="github"><diagram>UzV2zq1wL0osyPDNT0nNUTV2VTV2LsrPL4GwciucU3NyVI0MMlNUjV1UjYwMgFjVyA2HrCFY1qAgsSg1rwSLBiADYTaQg2Y1AA==</diagram></mxfile>
```

Comment:
What is this file for? Also, RemarkModelDiagram is blank (and doesn't seem to be part of this command).

Reply:
I've removed the diagram which is meant for the remark command. Also, I don't really understand the inefficient implementation I've made, could we discuss this on this coming wed meeting instead?